interpreters/src/table_manipulator/meta_based.rs

```diff
@@ -105,17 +101,21 @@ impl TableManipulator for TableManipulatorImpl {
         plan: DropTablePlan,
         _table_engine: TableEngineRef,
     ) -> Result<Output> {
+        let partition_table_info =
```

Comment:
create_to_partition_table_info?

Reply:
I rename it to `create_partition_table_info`.